safe_gpu/safe_gpu.py

```diff
@@ -140,13 +150,52 @@ def allocate_gpus(self, gpu_device_numbers):
 
         time.sleep(self.debug_sleep)
 
+        if not self.placeholder_fn:
```

Comment:
Why would we allow no `placeholder_fn`? Most objects that evaluate as False (`None`, empty containers, zero) are not callable and will crash at line 157 anyway.

Reply:
The idea was, if there is a case where no placeholder is possible, 
the `None` can be a solution, disabling placeholder mechanism.

But I don't need that case, coz i have solved it with pycuda...